public/app/features/datasources/settings/HttpSettingsCtrl.ts

```diff
@@ -1,5 +1,5 @@
 import { coreModule } from 'app/core/core';
-import { createChangeHandler, createResetHandler, PasswordFieldEnum } from '../utils/passwordHandlers';
+// import { createChangeHandler, createResetHandler, PasswordFieldEnum } from '../utils/passwordHandlers';
```

Comment:
```suggestion
```

Reply:
should be able to remove other commented code in this file too?

Reply:
yes